pkg/backup/item_backupper_test.go

```diff
@@ -106,6 +106,35 @@ func TestBackupItemSkips(t *testing.T) {
 	}
 }
 
+func TestExcludingCompletedPods(t *testing.T) {
+	tests := []struct {
+		name  string
+		phase string
+	}{
+		{
+			name:  "Failed pods aren't backed up",
+			phase: "Failed",
+		},
+		{
+			name:  "Succeeded pods aren't backed up",
+			phase: "Succeeded",
+		},
+	}
+
+	for _, test := range tests {
```

Comment:
@skriss what would you recommend for test logic here? I was thinking about asserting that no actions were called, since there's a few cases where `backupItem` could return nil, but I don't see a mock in the code base that would do that already.

Reply:
I'm thinking you probably want to add these as cases to `TestBackupItemSkips` which already has the test logic set up. I'd just confirm that adding the test cases fail before you make the code change, and pass after.